### PR TITLE
Create types.rst

### PR DESCRIPTION
**Pull request by jamesray1**
Added a link for more information about operators.

### CODE REVIEW

**Comment by pirapira** [comment]
@jamesray1 can you title the four PRs differently?  That will help to find which is which.

File: docs/types.rst [hunk 1]
```diff
@@ -41,6 +41,8 @@ Operators:
 
 The operators ``||`` and ``&&`` apply the common short-circuiting rules. This means that in the expression ``f(x) || g(y)``, if ``f(x)`` evaluates to ``true``, ``g(y)`` will not be evaluated even if it may have side-effects.
 
+For more information about logical and relational operators (as used in C++), see `[here](http://www.learncpp.com/cpp-tutorial/36-logical-operators/)`_ and `[here](http://www.learncpp.com/cpp-tutorial/35-relational-operators-comparisons/)`_, respectively.
```
Comment (pirapira) [inline docs/types.rst]: These markups look like a combination of Markdown and reStructuredText.

**Review by pirapira — DISMISSED** [review]
Please fix the link format.  See [reStructuredText reference](http://docutils.sourceforge.net/docs/user/rst/quickref.html).

**Comment by jamesray1** [comment]
@pirapira done, please see my revised commit. The links preview correctly now.

**Comment by axic** [comment]
Can you please use more meaningful commit messages?

**Comment by jamesray1** [comment]
@axic OK, will do. Although, it is a fairly minor revision commit, and you can see the details by clicking on the file.

**Comment by axic** [comment]
@jamesray1 I understand that, but by looking at the git history, one would be very confused seeing a couple of commits titled "Create types.rst".

**Comment by pirapira** [comment]
@jamesray1 if you don't mind, I will create a very similar pull-request with modified commit messages.

**Comment by jamesray1** [comment]
Please see an updated commit: https://github.com/ethereum/solidity/pull/2406/commits/208454382f46c6ef1f4cfd14406cf24888b7760e

**Pull request closed without merging**